dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -151,13 +151,12 @@ boolean isSystemClass(String className) {
    *
    * @param jarFilePath absolute path to a jar file
    */
-  static SymbolReferenceSet scanSymbolReferencesInJar(Path jarFilePath)
-      throws IOException {
+  SymbolReferenceSet scanSymbolReferencesInJar(Path jarFilePath) throws IOException {
     checkArgument(jarFilePath.isAbsolute(), "The input jar file path is not an absolute path");
     checkArgument(Files.isReadable(jarFilePath), "The input jar file path is not readable");
 
     SymbolReferenceSet.Builder symbolTableBuilder = SymbolReferenceSet.builder();
-    for (JavaClass javaClass : listClassesInJar(jarFilePath)) {
+    for (JavaClass javaClass : listClassesInJarFromClassPath(jarFilePath)) {
```

Comment:
Because `listClassesInJarFromClassPath` uses instance variable `classRepository`, these methods are now non-static method.

Reply:
Right. That design works well.